### ISSUE TITLE
group email doesn't work from .email.to('email1, email2')

### ISSUE THREAD

**Issue by atveretinov** [influxdata/kapacitor#830]
Hi,

It seems the group email is not working.

.email().to('email-to_1@mail.com, email-to_2@mail.com')

The person email-to_1 receives this email and sees the email of "email-to_2" in the TO field, but person email-to_2 will never  receive it.

Is it bug or just feature and we cannot send emails like that?

Thank you.
Alexander.

**Comment by nathanielc** [comment]
@atveretinov Try this instead:

```
.email().to('email-to_1@mail.com', 'email-to_2@mail.com')
```

**Comment by superbool** [comment]
@nathanielc @atveretinov 
When I use 
```
.email().to('email-to_1@mail.com', 'email-to_2@mail.com')
``` 
the kapacitor will get the the error log,but when I use 
```
.email()('email-to_1@mail.com', 'email-to_2@mail.com')
```
everything is ok,  **.to** is no need.

```2016/08/25 09:22:39 http: panic serving 172.17.4.151:52018: reflect: call of reflect.Value.Interface on zero Value
goroutine 1487 [running]:
net/http.(*conn).serve.func1(0xc820276580)
       	/usr/local/go/src/net/http/server.go:1389 +0xc1
panic(0xc6a160, 0xc82030e9a0)
       	/usr/local/go/src/runtime/panic.go:443 +0x4e9
github.com/influxdata/kapacitor/tick.Evaluate.func1(0xc8201dd3b0)
       	/root/go/src/github.com/influxdata/kapacitor/tick/eval.go:65 +0x323
panic(0xc6a160, 0xc82030e9a0)
       	/usr/local/go/src/runtime/panic.go:443 +0x4e9
reflect.valueInterface(0x0, 0x0, 0x0, 0xc8203b2001, 0x0, 0x0)
       	/usr/local/go/src/reflect/value.go:913 +0x7f
reflect.Value.Interface(0x0, 0x0, 0x0, 0x0, 0x0)
       	/usr/local/go/src/reflect/value.go:908 +0x48
github.com/influxdata/kapacitor/tick.(*ReflectionDescriber).Property(0xc8203be030, 0xc82015f8d5, 0x5, 0x0, 0x0)
       	/root/go/src/github.com/influxdata/kapacitor/tick/eval.go:641 +0xb9
github.com/influxdata/kapacitor/tick.evalChain(0x7f12dbc509a0, 0xc820116d70, 0xc8201dd390, 0xc8203b2c18, 0x0, 0x0)
       	/root/go/src/github.com/influxdata/kapacitor/tick/eval.go:289 +0x2e6
github.com/influxdata/kapacitor/tick.eval(0x7f12dbc50718, 0xc820116d70, 0xc8201dd390, 0xc8203b2c18, 0x0, 0x0)
       	/root/go/src/github.com/influxdata/kapacitor/tick/eval.go:156 +0x3b9
github.com/influxdata/kapacitor/tick.eval(0x7f12dbc50718, 0xc820117220, 0xc8201dd390, 0xc8203b2c18, 0x0, 0x0)
       	/root/go/src/github.com/influxdata/kapacitor/tick/eval.go:148 +0x2c6
github.com/influxdata/kapacitor/tick.eval(0x7f12dbc50718, 0xc820117360, 0xc8201dd390, 0xc8203b2c18, 0x0, 0x0)
       	/root/go/src/github.com/influxdata/kapacitor/tick/eval.go:148 +0x2c6
github.com/influxdata/kapacitor/tick.eval(0x7f12dbc50958, 0xc820622e70, 0xc8201dd390, 0xc8203b2c18, 0x0, 0x0)
       	/root/go/src/github.com/influxdata/kapacitor/tick/eval.go:191 +0x1789
github.com/influxdata/kapacitor/tick.Evaluate(0xc820371500, 0x168, 0xc8201dd390, 0x0, 0x0)
       	/root/go/src/github.com/influxdata/kapacitor/tick/eval.go:77 +0x146
github.com/influxdata/kapacitor/pipeline.CreatePipeline(0xc820371500, 0x168, 0x1, 0xc8201dd390, 0x7f12dbc1ea38, 0xc820183180, 0xc82030e240, 0x0, 0x0)
       	/root/go/src/github.com/influxdata/kapacitor/pipeline/pipeline.go:49 +0x229
github.com/influxdata/kapacitor.(*TaskMaster).NewTask(0xc820066a80, 0xc8201dd360, 0xa, 0xc820371500, 0x168, 0x0, 0xc82030e240, 0x1, 0x1, 0xdf8475800, ...)
       	/root/go/src/github.com/influxdata/kapacitor/task_master.go:247 +0x14f
github.com/influxdata/kapacitor/services/task_store.(*Service).newKapacitorTask(0xc8200b4750, 0xc8201dd360, 0xa, 0x0, 0xc82030e220, 0x1, 0x1, 0xc820371500, 0x168, 0x0, ...)
       	/root/go/src/github.com/influxdata/kapacitor/services/task_store/service.go:937 +0x2a9
github.com/influxdata/kapacitor/services/task_store.(*Service).handleCreateTask(0xc8200b4750, 0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/root/go/src/github.com/influxdata/kapacitor/services/task_store/service.go:738 +0x934
github.com/influxdata/kapacitor/services/task_store.(*Service).(github.com/influxdata/kapacitor/services/task_store.handleCreateTask)-fm(0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/root/go/src/github.com/influxdata/kapacitor/services/task_store/service.go:128 +0x3e
net/http.HandlerFunc.ServeHTTP(0xc82000ecb0, 0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/usr/local/go/src/net/http/server.go:1618 +0x3a
github.com/influxdata/kapacitor/services/httpd.jsonContent.func1(0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/root/go/src/github.com/influxdata/kapacitor/services/httpd/handler.go:555 +0xaa
net/http.HandlerFunc.ServeHTTP(0xc820011600, 0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/usr/local/go/src/net/http/server.go:1618 +0x3a
github.com/influxdata/kapacitor/services/httpd.gzipFilter.func1(0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/root/go/src/github.com/influxdata/kapacitor/services/httpd/handler.go:541 +0xda
net/http.HandlerFunc.ServeHTTP(0xc820011620, 0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/usr/local/go/src/net/http/server.go:1618 +0x3a
github.com/influxdata/kapacitor/services/httpd.versionHeader.func1(0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/root/go/src/github.com/influxdata/kapacitor/services/httpd/handler.go:564 +0xc0
net/http.HandlerFunc.ServeHTTP(0xc820011640, 0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/usr/local/go/src/net/http/server.go:1618 +0x3a
github.com/influxdata/kapacitor/services/httpd.cors.func1(0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/root/go/src/github.com/influxdata/kapacitor/services/httpd/handler.go:597 +0x329
net/http.HandlerFunc.ServeHTTP(0xc820011660, 0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/usr/local/go/src/net/http/server.go:1618 +0x3a
github.com/influxdata/kapacitor/services/httpd.requestID.func1(0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/root/go/src/github.com/influxdata/kapacitor/services/httpd/handler.go:607 +0x15d
net/http.HandlerFunc.ServeHTTP(0xc820011680, 0x7f12dbc5bf70, 0xc82030e0c0, 0xc8204cc460)
       	/usr/local/go/src/net/http/server.go:1618 +0x3a
github.com/influxdata/kapacitor/services/httpd.logging.func1(0x7f12dbc5bf70, 0xc82030e080, 0xc8204cc460)
       	/root/go/src/github.com/influxdata/kapacitor/services/httpd/handler.go:615 +0x10b
net/http.HandlerFunc.ServeHTTP(0xc8200116a0, 0x7f12dbc5bf70, 0xc82030e080, 0xc8204cc460)
       	/usr/local/go/src/net/http/server.go:1618 +0x3a
github.com/influxdata/kapacitor/services/httpd.recovery.func1(0x7f12dbc5bf10, 0xc82062a000, 0xc8204cc460)
       	/root/go/src/github.com/influxdata/kapacitor/services/httpd/handler.go:625 +0x114
net/http.HandlerFunc.ServeHTTP(0xc8200116c0, 0x7f12dbc5bf10, 0xc82062a000, 0xc8204cc460)
       	/usr/local/go/src/net/http/server.go:1618 +0x3a
github.com/influxdata/kapacitor/services/httpd.(*ServeMux).ServeHTTP(0xc8201d35f0, 0x7f12dbc5bf10, 0xc82062a000, 0xc8204cc460)
       	/root/go/src/github.com/influxdata/kapacitor/services/httpd/mux.go:163 +0x17d
github.com/influxdata/kapacitor/services/httpd.(*Handler).ServeHTTP(0xc820074f00, 0x7f12dbc5bf10, 0xc82062a000, 0xc8204cc460)
       	/root/go/src/github.com/influxdata/kapacitor/services/httpd/handler.go:295 +0xe5
net/http.serverHandler.ServeHTTP(0xc820162400, 0x7f12dbc5bf10, 0xc82062a000, 0xc8204cc460)
       	/usr/local/go/src/net/http/server.go:2081 +0x19e
net/http.(*conn).serve(0xc820276580)
       	/usr/local/go/src/net/http/server.go:1472 +0xf2e
created by net/http.(*Server).Serve
       	/usr/local/go/src/net/http/server.go:2137 +0x44e
```

**Comment by nathanielc** [comment]
@superbool That panic with `.to` was fixed in the 1.0 beta versions. My guess is you are using 0.13.1.

**Comment by superbool** [comment]
Yes ,I'm using 0.13.1.
